### ISSUE TITLE
LFG Search taint

### ISSUE THREAD

**Issue by Flowy** [arkayenro/arkinventory#1773]
**What version number you are using?**
current and old

**What game client are you playing? Retail, Wrath, Classic / Live, PTR, Beta**
wrath

**What language is the game client set to?**
en

**Describe the bug**
sometime when opening LFG tool, ArkInventory gets blamed and search does not work until reload
21x [ADDON_ACTION_BLOCKED] AddOn 'ArkInventory' tried to call the protected function 'Search()'.

**To Reproduce**
I don't know what exactly causes it, but similar thing happened for LootReserve and it got fixed

Anonomit — Yesterday at 11:20 PM

doing anything related to UIDropDownMenu causes taint that blocks group finder from using its protected Search function. the "solution" is to not use the blizzard-provided template or any functions or variables related to UIDropDownMenu. this library (https://www.wowace.com/projects/libuidropdownmenu) explains the issue a bit, and recreates all UIDropDownMenu related functions and variables for use by non-blizzard addons. unfortunately, all frames that inherit UIDropDownMenuTemplate will need to be created with lua instead of xml to fix the taint

**Screenshots**
21x [ADDON_ACTION_BLOCKED] AddOn 'ArkInventory' tried to call the protected function 'Search()'.

**Comment by arkayenro** [comment]
i dont use uidropdownmenu at all in my code. so you will need to check the full stack trace, or enable the taint log, to work out where its coming from. see https://github.com/arkayenro/arkinventory/wiki/Taint-and-ADDON_ACTION_BLOCKED

**Comment by Flowy** [comment]
will this help?

1/29 21:04:37.049  An action was blocked because of taint from ArkInventory - Search()
1/29 21:04:37.049      Interface/AddOns/Blizzard_LookingForGroupUI/Blizzard_LFGBrowse.lua:226 LFGBrowse_DoSearch()
1/29 21:04:37.049      Interface/AddOns/Blizzard_LookingForGroupUI/Blizzard_LFGBrowse.lua:179 LFGBrowseFrame:SearchActiveEntry()
1/29 21:04:37.049      Interface/AddOns/Blizzard_LookingForGroupUI/Blizzard_LFGParentFrame.lua:92 LFGParentFrame_SearchActiveEntry()
1/29 21:04:37.049      Interface/AddOns/Blizzard_LookingForGroupUI/Blizzard_LFGListing.lua:69

**Comment by arkayenro** [comment]
nope.  sorry, youre going to have to find the source of the taint

99% sure it wont be arkinventory (as i dont use the uidropdownmenu, i use my own menu library)
